int16 id
int64 xmin
int64 ymin
int64 xmax
int64 ymax
int16 conf
int16 cls
geometry_msgs/Vector3 position_from_realsense
